tests/issues.rs

```diff
@@ -105,3 +105,51 @@ mod issue514 {
         assert_eq!(reader.read_event().unwrap(), Event::Eof);
     }
 }
+
+#[test]
+fn issue_604_short() {
```

Comment:
Please put all tests in a module, like as above. Prefix in names not needed because they will be in a module